test/spec/modules/pubmaticBidAdapter_spec.js

```diff
@@ -978,6 +1033,15 @@ describe('PubMatic adapter', function () {
         expect(data.imp[1]['video']['w']).to.equal(multipleMediaRequests[1].mediaTypes.video.playerSize[0]);
         expect(data.imp[1]['video']['h']).to.equal(multipleMediaRequests[1].mediaTypes.video.playerSize[1]);
       });
+
```

Comment:
we need a case to handle native response

Reply:
done